### PR TITLE
[ENH] Derive masks to support native-space data

### PR DESCRIPTION
**Pull request by tsalo**
References #6.

Changes proposed in this pull request:

- Add function to derive masks from EPI data and CSF tissue probability map or mask.
- Have the ICA function return the relevant ICA outputs instead of inferring the filenames in the main workflow function.
- Use the new derived masks throughout the workflow.

### CODE REVIEW

File: aroma/utils.py [hunk 1]
```diff
@@ -6,6 +6,40 @@
...
+    else:
+        brain_img = masking.compute_epi_mask(in_file)
+        out_img = image.math_img("1 - mask", mask=brain_img)
+        csf_img = nib.load(csf)
+        csf_data = csf_img.get_fdata()
+        if len(np.unique(csf_data)) == 2:
+            print("CSF mask provided. Inferring other masks.")
+        else:
+            print("CSF TPM provided. Inferring CSF and other masks.")
+            csf_img = image.math_img("csf >= 0.95", csf=csf_img)
```
Comment (tsalo) [inline aroma/utils.py:45]: Per the ICA-AROMA paper:

> To this end we defined a CSF mask by thresholding a CSF segmentation prior, supplied as part of FSL, at 95% of the robust range.

@CesarCaballeroGaudes does that look good to you?

Comment (CesarCaballeroGaudes) [reply aroma/utils.py:45]: Yes, 95% looks good to me, and even 90%

Comment (CesarCaballeroGaudes) [reply aroma/utils.py:45]: Yes, 95% looks good. Even 90% would also be fine.

**Comment by tsalo** [comment]
Closing now that https://github.com/Brainhack-Donostia/aroma/pull/24 is open.

**Pull request closed without merging**